crates/swc_isograph_plugin/transform/tests/transform.rs

```diff
@@ -0,0 +1,53 @@
+use isograph_config::IsographProjectConfig;
+use std::{
+    fs::read_to_string,
+    path::{Path, PathBuf},
+};
+use swc_ecma_parser::{EsConfig, Syntax};
+use swc_ecma_transforms_testing::{test_fixture, FixtureTestConfig};
+use swc_isograph::compile_iso_literal_visitor;
+
+#[testing::fixture("tests/fixtures/base/**/*/input.js")]
```

Comment:
WHOA this is amazing, we have to rewrite our other tests to use this, I like it a lot more than the hand-written stuff I have